swig/python/flamegpu.i

```diff
@@ -313,12 +313,61 @@ class FLAMEGPURuntimeException : public std::exception {
     std::string str_str;
 };
 %}
-
+%inline %{
+    #include <sstream>
+    // Python include for stacktrace frames pre Python 3.11
+    #if PY_VERSION_HEX < 0x030900B1
```

Comment:
The macro is actually checking LT 3.9.0b1, not 3.11 (though this is the version that is 3.11 friendly) so the comment is maybe not the most clear but not worth the change.

Reply:
Got that hex thing from the py 3.11 changelog.